Vagrantfile

```diff
@@ -47,6 +47,13 @@ Vagrant.configure(2) do |config|
     vb.cpus = 2
   end
 
+  host_user = ENV.fetch("USER", "vagrant")
+  aws_profile = ENV.fetch("RF_AWS_PROFILE", "raster-foundry")
+  rf_aws_config_bucket = ENV.fetch("RF_AWS_CONFIG_BUCKET",
```

Comment:
Updated in https://github.com/azavea/raster-foundry/pull/1269/commits/3dcf90a87538b444ac6fd9a9e0809ffb916e1d69